client/src/components/Board/Settings/MembersList.js

```diff
@@ -0,0 +1,204 @@
+      url: SERVER_URL + "/board/member/86",
+      method: 'delete',
+      data: {member: memberToBeDelete._id}
+    });  
+    fetchBoardMembers();
+  }
+  catch(error){
+    const { isSuccess, message } = error.response.data;
+    if (!isSuccess) {
+      setShowLoader(false);
+      openSnackBar('error', message);
+    }
+  }
+  }
+
+  const openConfirmationDialog = (member) => {
+    setMemberToBeDelete(member);
+    setOpenConfirmationModal(true);
+  }
+
+  const closeConfirmationModal = () => {
+    setOpenConfirmationModal(false);
+  }
+  return showLoader ? (
+    <CircularProgress className="loader-center" />
+  ) : (
+    <div className="member-table">
+      <Paper>
+        <Typography variant="h6" id="tableTitle" className="table-heading">
+          Board Members
+        </Typography>
+        <Table>
+          <TableHead>
+            <TableRow>
+              <TableCell>Name</TableCell>
+              <TableCell align="center">Role</TableCell>
+              <TableCell align="center">Update Role</TableCell>
+              <TableCell align="center">Actions</TableCell>
+            </TableRow>
+          </TableHead>
+          <TableBody>
+            {members.map((row, index) => {
+              if (row.role === constants.ROLES.ADMIN) {
+                row.switch = true;
+              } else {
+                row.switch = false;
+              }
+              return (
+                <TableRow key={index}>
+                  <TableCell component="th" scope="row">
+                    {row.user.name}
+                  </TableCell>
+                  <TableCell align="center">
+                    <Chip label={row.role} color="primary" />
+                  </TableCell>
+                  <TableCell align="center">
+                    {row.role !== constants.ROLES.SUPER_ADMIN && (
+                      <Switch
+                        checked={row.switch}
+                        onChange={event => {
+                          setMemberRole(event.target.checked, index);
+                        }}
+                        value="roleSwitch"
+                        color="primary"
+                        inputProps={{ "aria-label": "primary checkbox" }}
+                      />
+                    )}
+                  </TableCell>
+                  <TableCell align="center">
+                  {row.role !== constants.ROLES.SUPER_ADMIN && (
+                      <Icon style={{cursor: 'pointer'}} onClick={()=>{
+                        openConfirmationDialog(row.user);
+                      }}>delete</Icon>
+                  )}
+                  </TableCell>
+                </TableRow>
+              );
+            })}
+          </TableBody>
+        </Table>
+      </Paper>
+
+      <DeleteModal user={memberToBeDelete} 
+              openModal={openConfirmationModal} 
+              onClose={closeConfirmationModal}
+              onConfirm={handleMemberDelete}
+              />
```

Comment:
Correct the indentation and remove extra lines.

Use Prettier on Save Or manually.